app/src/main/java/de/westnordost/streetcomplete/quests/crossing_markings/AddCrossingMarkings.kt

```diff
@@ -11,40 +11,38 @@ import de.westnordost.streetcomplete.osm.Tags
 import de.westnordost.streetcomplete.osm.isCrossing
 import de.westnordost.streetcomplete.osm.updateCheckDateForKey
 import de.westnordost.streetcomplete.osm.updateWithCheckDate
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
 
-class AddCrossingType : OsmElementQuestType<CrossingType> {
+class AddCrossingMarkings : OsmElementQuestType<Boolean> {
 
-    /*
-       Always ask for deprecated/meaningless values (island, unknown, yes)
-
-       Only ask again for crossing types that are known to this quest so to be conservative with
-       existing data
-     */
     private val crossingFilter by lazy { """
-        nodes with highway = crossing
+        nodes with
+          highway = crossing
           and foot != no
-          and (
-            !crossing
-            or crossing ~ island|unknown|yes
-            or (
-              crossing ~ traffic_signals|uncontrolled|zebra|marked|unmarked
-              and crossing older today -8 years
-            )
-          )
+          and !crossing:markings
+          and (!crossing or crossing = island)
+          and crossing:signals != yes
     """.toElementFilterExpression() }
+    /* only looking for crossings that have no crossing=* at all set because if the crossing was
+     * - if it had markings, it would be tagged with "marked","zebra" or "uncontrolled"
+     * - if it hadn't, it would be tagged with "unmarked"
+     * - and in case of "traffic_signals", we currently assume that when there are traffic signals
+     *   it would be spammy to ask about markings because the answer would almost always be "yes".
+     *   Might differ per country, research necessary. */
```

Comment:
I realize you aren’t keen on _applying_ `crossing:signals=separate` or `shared`, especially since [this proposal](https://wiki.openstreetmap.org/wiki/Proposal:Crossing_signalization) is still a draft. However, if your intention is to exclude signalized crossings, then the filter would be more effective if it also excludes the [1,412 nodes](https://taginfo.openstreetmap.org/keys/crossing%3Asignals?filter=nodes#values) with these values.

I agree that the “spamminess” differs markedly by country. Hopefully there will be some flexibility in the future to ask about markings for some obvious regions like the U.S. (outside of the largest cities perhaps). But even if not, markings are much easier to armchair-map than signalization.

Reply:
Right, changed it to `and (!crossing:signals or crossing:signals = no)`